EIPS/eip-888.md

```diff
@@ -0,0 +1,178 @@
+---
+eip: 888
+title: EXP Token Standard
```

Comment:
Personally, I think it would be better to be more verbose in EIPs to be thoughtful of non-native English speakers.

Reply:
Thanks for the discussion at the EIPIP meeting yesterday. Based on that discussion, I'll let it for now and add clarifying notes in the abstract and rationale that EXP is shorthand for "experience points".